site/content/docs/alerting/ssl-expiration.md

```diff
@@ -28,6 +28,12 @@ Simply create or pick an existing alert channel that your check subscribes to an
 ## API checks
 The domain for the certificate is parsed from the `URL` in the HTTP request settings so it does not require any setup.
 
+{{<info >}}
```

Comment:
Do we have alternatives for highlighting errors other than <info>? Is there another type of callout that would fit this better and graphically distinguish common error callouts from generic info callouts?

Reply:
We could use quote blocks? 
![image](https://user-images.githubusercontent.com/26063772/145892482-7cbbd6de-9582-420a-85d4-a49bf6879774.png)